model/beer.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const uuidv1 = require('node-uuid').v1;
+const storage = require('../lib.storage.js');
```

Comment:
Your path to the storage module isn't correctly written. Look at it again and figure out what you're missing.